modules/jaeger-removing-instance-cli.adoc

```diff
@@ -11,7 +11,7 @@ This module included in the following assemblies:
 +
 [source,terminal]
 ----
-$ oc login
+$ oc login --username=<NAMEOFUSER> 
```

Comment:
should this contain the `https://<HOSTNAME>:8443` part too?

Reply:
This file was removed as part of work Julie performed for another story, and is no longer part of this effort.